docs/getting-started-guides/scratch.md

```diff
@@ -582,45 +561,40 @@ If you are using the HTTPS approach, then set:
 }
 ```
 
-The `/etc/ssl` mount allows the apiserver to find the SSL root certs so it can
-authenticate external services, such as a cloud provider.
-
-The `/srv/kubernetes` mount allows the apiserver to read certs and credentials stored on the
-node disk.
-
-Optionally, you may want to mount `/var/log` as well and redirect output there.
-
-#### Starting Apiserver
-Place the completed pod template into the kubelet config dir
-(whatever `--config=` argument of kubelet is set to, typically
-`/etc/kubernetes/manifests`).
-
-Next, verify that kubelet has started a container for the apiserver:
-```
-$ sudo docker ps | grep apiserver:
-5783290746d5        gcr.io/google_containers/kube-apiserver:e36bf367342b5a80d7467fd7611ad873            "/bin/sh -c '/usr/lo'"    10 seconds ago      Up 9 seconds                              k8s_kube-apiserver.feb145e7_kube-apiserver-kubernetes-master_default_eaebc600cf80dae59902b44225f2fc0a_225a4695   ```
-```
-
-Then try to connect to the apiserver:
-```
-$ echo $(curl -s http://localhost:8080/healthz)
-ok
-$ curl -s http://localhost:8080/api
-{
-  "versions": [
-    "v1beta3",
-    "v1"
-  ]
-}
-```
-
-If you have selected the `--register-node=true` option for kubelets, they will now being self-registering with the apiserver.
-You should soon be able to see all your nodes by running the `kubect get nodes` command.
-Otherwise, you will need to manually create node objects.
 
-### Scheduler
+Here are some apiserver flags you may need to set:
+  - `--cloud-provider=`
+  - `--cloud-config=` if cloud provider requires a config file (GCE, AWS). If so, need to put config file into apiserver image or mount through hostPath.
```

Comment:
Revise (as suggested below) and also provide/insert more about what or where the "config file" is or comes from (is this kubeconfig? if yes, we should clarify):

... set if the cloud provider (GCE, AWS), requires a config file. You must place the config file into the apiserver image or mount it by specifying hostPath.